src/pages/mesh/advanced/caching/index.md

```diff
@@ -29,6 +29,37 @@ If you want to cache API Mesh responses, you have two options:
 
 Regardless of the option you choose, you can configure your [cache-control headers](./cache-control-headers.md) to control how long a response is cached.
 
+## Caching requirements
+
+For API Mesh to cache a response, the request and response must meet the following requirements:
+
+**Request**
+
+- The request method must be `GET` or `POST`.
+- The request must be a query operation type.
+- The request must not be an introspection query.
+
+**Response**
+
+- The response status must be in the range `200` - `299`.
+- The response body cannot contain errors, as defined by [GraphQL specification](https://spec.graphql.org/October2021/#sec-Errors).
+- The response `cache-control` header must contain public [cache eligible directives](./cache-control-headers.md#response-headers).
+
+## Compliance
+
+When considering PCI or HIPPA compliance as it relates to caching, you should understand the following:
+
+- API Mesh caching is disabled by default.
+  - You must opt in to caching, by configuring `meshConfig.responseConfig.cache: true`.
+- When caching is configured, API Mesh acts as a public cache driven by cache-control headers returned from your sources.
+- API Mesh is a gateway and is unaware of the compliance requirements of your sources. You should understand your sources and ensure they return appropriate cache-control headers to prevent caching of sensitive data.
+
+## Source-driven caching
+
+API Mesh implements a source-driven caching model. Your data sources are responsible for directing caching behavior by returning appropriate [cache-control headers](./cache-control-headers.md). You must ensure that sources serving personalized content return appropriate cache-control directives. API Mesh will never cache responses containing the `private` or `no-store` directives.
```

Comment:
Thoughts on using an inline alert to really emphasize the `You must ensure that sources  serving personalized content return appropriate cache-control directives`?